lib/actors/examples/fannkuchredux/fannkuchredux.nit

```diff
@@ -0,0 +1,211 @@
+# This file is part of NIT (http://www.nitlanguage.org).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example implemented from "The computer Language Benchmarks Game" - Fannkuch-Redux
+# http://benchmarksgame.alioth.debian.org/
+#
+# Complete description of the fannkuch-redux :
+# https://benchmarksgame.alioth.debian.org/u64q/fannkuchredux-description.html#fannkuchredux
+module fannkuchredux is no_warning("missing-doc")
+
+import actors
+
+class FannkuchRedux
+	actor
+
+	var p: Array[Int] is noautoinit
+	var pp: Array[Int] is noautoinit
+	var count: Array[Int] is noautoinit
+
+	fun print_p do
+		for i in [0..p.length[ do printn p[i] + 1
+		print ""
+	end
+
+	fun first_permutation(idx: Int) do
+		for i in [0..p.length[ do p[i] = i
+
+		var i = count.length - 1
+		while i > 0 do
```

Comment:
Est-ce possible de faire une boucle à l'envers en nit --  au lieu d'un while:

for i in [count.length-1..0].step(-1) do